### ISSUE TITLE
Nested bytestring groups throw a ParseTypeError

### ISSUE THREAD

**Issue by ricardo-dias** [plyara/plyara#30]
Example:
```
rule sample {
    strings:
        $ = { 4D 5A ( 90 ( 00 | 01 ) | 89 ) }
    condition:
        all of them
}
```
Throws: plyara.ParseTypeError: Illegal bytestring character ( at line 4

**Comment by Taskr** [comment]
submitted #31 to account for this. Should be reviewed in the near future. Thanks for submitting this @ricardo-dias.

**Comment by rshipp** [comment]
Thanks for the issue! I've pushed @Taskr's fix to PyPI as v1.3.2. Feel free to let us know if you find anything else.

**Comment by ricardo-dias** [comment]
It's working like a charm. Thanks!

**Comment by ricardo-dias** [comment]
> Thanks for the issue! I've pushed @Taskr's fix to PyPI as v1.3.2. Feel free to let us know if you find anything else.

It's working like a charm. Thanks!
